src/components/LessonForm.jsx

```diff
@@ -117,6 +119,12 @@ export default function LessonForm({lessonMaterials, onSubmit}) {
                     name="exitTicketPrompt"
                   />
 
+                  <Form.Group>
+                    <Form.Label>
+                      Have any feedback? Sumbit it <a href={FEEDBACK_URL} rel="noopener noreferrer" target="_blank">here</a>        
+                    </Form.Label>
+                  </Form.Group>
```

Comment:
Would this make more sense in the left column, right below the links to the materials?

Reply:
Of course one could argue that here is better because it is harder to miss

Reply:
Yeah I was going back and forth on this. I feel like it stands out a bit more here. It seems to get lost in the other links though I could be convinced otherwise.